### DISCUSSION TITLE
[Question]: Is there a way to detect camera comm malfunction?

### DISCUSSION THREAD

**Discussion by ryosoftware** [blakeblackshear/frigate#12370] [Ask A Question] [answered]
### What is your question:

Hello,
Sometimes cameras disconnects from Frigate and a black screen with a text is displayed instead of the camera image wthen connect to the Frigate UI.
Is there a way to detect this situation via a API query like /api/stats or anytrhing else?
Thanks

**Comment by NickM-27** [comment]
Yes, via MQTT or /api/stats the camera_fps will be 0

Also I'm home assistant this will present as the camera being unavailable

**Comment by ryosoftware** [comment]
Only to confirm. The expected value if error is 0 or 0.0
Thanks.

**Reply by NickM-27** [reply, marked as answer]
should be 0
